blueoil/cmd/init.py

```diff
@@ -131,7 +129,11 @@
     ("cosine", "'cosine' -> cosine learning rate."),
     ("2-step-decay", "'2-step-decay' -> learning rate decrease by 1/10 on {epochs}/2 and {epochs}-1."),
     ("3-step-decay", "'3-step-decay' -> learning rate decrease by 1/10 on {epochs}/3 and {epochs}*2/3 and {epochs}-1"),
-    ("3-step-decay-with-warmup", "'3-step-decay-with-warmup' -> warmup learning rate 1/1000 in first epoch, then train the same way as '3-step-decay'"),
+    (
+     "3-step-decay-with-warmup",
+     "'3-step-decay-with-warmup' -> \
+      warmup learning rate 1/1000 in first epoch, then train the same way as '3-step-decay'"
+    ),
```

Comment:
This backslash doesn't work because it's included in a string. And as @odoku -san mentioned, using backslashes for wrapping long lines is not recommended.

In this case, code like this would be fine.
```python
(
    "3-step-decay-with-warmup",
    "'3-step-decay-with-warmup' -> "
    "warmup learning rate 1/1000 in first epoch, then train the same way as '3-step-decay'"
),
```

Reply:
Wow I see.

Reply:
@tfujiwar Changed with 1577c22